app/src/sharedTest/java/org/oppia/app/profile/AdminPinActivityTest.kt

```diff
@@ -148,12 +150,12 @@ class AdminPinActivityTest {
   }
 
   @Test
-  fun testAdminPinActivity_configurationChange_inputShortPin_clickSubmit_checkPinLengthError() {
+  fun testAdminPinActivity_configurationChange_inputShortPin_clickSubmit_clickIsDisabled() {
     ActivityScenario.launch<AdminPinActivity>(AdminPinActivity.createAdminPinActivityIntent(context, 0, -10710042, 0)).use {
```

Comment:
You can use launch directly importing ActivityScenario.

Reply:
Updated.